src/devcenter/azext_devcenter/custom.py

```diff
@@ -0,0 +1,1034 @@
+    List as _CatalogList,
+    Show as _CatalogShow,
+    Sync as _CatalogSync,
+    Update as _CatalogUpdate,
+    Wait as _CatalogWait,
+)
+from .aaz.latest.devcenter.admin.devbox_definition import (
+    Create as _DevBoxDefinitionCreate,
+    Delete as _DevBoxDefinitionDelete,
+    List as _DevBoxDefinitionList,
+    Show as _DevBoxDefinitionShow,
+    Update as _DevBoxDefinitionUpdate,
+    Wait as _DevBoxDefinitionWait,
+)
+from .aaz.latest.devcenter.admin.devcenter import List as _DevCenterList
+from .aaz.latest.devcenter.admin.environment_type import (
+    Create as _EnvironmentTypeCreate,
+    Delete as _EnvironmentTypeDelete,
+    List as _EnvironmentTypeList,
+    Show as _EnvironmentTypeShow,
+    Update as _EnvironmentTypeUpdate,
+)
+from .aaz.latest.devcenter.admin.gallery import (
+    Create as _GalleryCreate,
+    Delete as _GalleryDelete,
+    List as _GalleryList,
+    Show as _GalleryShow,
+    Wait as _GalleryWait,
+)
+from .aaz.latest.devcenter.admin.image import (
+    List as _ImageList,
+    Show as _ImageShow,
+)
+from .aaz.latest.devcenter.admin.image_verion import (
+    List as _ImageVersionList,
+    Show as _ImageVersionShow,
+)
+from .aaz.latest.devcenter.admin.network_connection import (
+    List as _NetworkConnectionList,
+    ListHealthDetail as _NetworkConnectionListHealthDetail,
+)
+from .aaz.latest.devcenter.admin.pool import (
+    Create as _PoolCreate,
+    Delete as _PoolDelete,
+    List as _PoolList,
+    Show as _PoolShow,
+    Update as _PoolUpdate,
+    Wait as _PoolWait,
+)
+from .aaz.latest.devcenter.admin.project import List as _ProjectList
+from .aaz.latest.devcenter.admin.project_allowed_environment_type import (
+    List as _ProjectAllowedEnvironmentTypeList,
+    Show as _ProjectAllowedEnvironmentTypeShow,
+)
+from .aaz.latest.devcenter.admin.project_environment_type import (
+    Create as _ProjectEnvironmentTypeCreate,
+    Delete as _ProjectEnvironmentTypeDelete,
+    List as _ProjectEnvironmentTypeList,
+    Show as _ProjectEnvironmentTypeShow,
+    Update as _ProjectEnvironmentTypeUpdate,
+)
+from .aaz.latest.devcenter.admin.schedule import (
+    Create as _ScheduleCreate,
+    Delete as _ScheduleDelete,
+    Show as _ScheduleShow,
+    Update as _ScheduleUpdate,
+    Wait as _ScheduleWait,
+)
+from .aaz.latest.devcenter.admin.sku import List as _SkuList
+from ._validators import validate_attached_network_or_dev_box_def
+
+# Control plane
+
+
+def set_configured_defaults(args):
+    for arg_name, arg in args:
+        if arg_name == "project_name":
+            arg.configured_default = "project"
+        if arg_name == "dev_center_name":
+            arg.configured_default = "dev-center"
+    return args
+
+
+class AttachedNetworkCreate(_AttachedNetworkCreate):
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        args_schema = super()._build_arguments_schema(*args, **kwargs)
+        args_schema.network_connection_id._required = True
```

Comment:
For those arguments which is not used in commands, you should hide them in aaz-dev-tools 
![image](https://user-images.githubusercontent.com/69238381/223006822-5b52181e-380f-476e-86b0-c4f4b62b4d64.png)